ui/src/app/utils/profile-utils.tsx

```diff
@@ -0,0 +1,15 @@
+import { Profile } from 'generated/fetch';
+
+import { Country, INTL_USER_SIGN_IN_CHECK } from 'app/utils/constants';
+import { authStore } from 'app/utils/stores';
+
+export const getProfilePictureSrc = () => {
+  return authStore.get().auth?.user?.profile.picture;
+};
+
+export const isUserFromUSAOrSignedInBeforeNov = (profile: Profile) => {
+  const userCountry = profile?.address?.country;
+  const signIn = new Date(profile.firstSignInTime);
+  const signed_in_after_nov = INTL_USER_SIGN_IN_CHECK <= signIn;
```

Comment:
1. This should be `signed_in_before_nov`
2. This should be `<`, not `<=`, to account for day-of users.

Reply:
Yes before my bad